sponsoring.tex

```diff
@@ -140,7 +140,7 @@ \section{Über Uns}
 
 In unserem Hackerspace möchten wir einerseits die Möglichkeiten und das Umfeld
 zur Umsetzung von interessanten technischen oder künstlerischen Projekten
-anbieten, andererseits auch den Austausch von Wissen fördern. Dies kann
+anbieten, andererseits aber auch den Austausch von Wissen fördern. Dies kann
```

Comment:
Das "aber" finde ich unnötig. Ich würde den Satz ganz umkrempeln:

In unserem Hackerspace möchten wir die Möglichkeit und das Umfeld bieten um interessante, technische sowie künstlerische Projekten umzusetzen. Ebenfalls wollen wir den Austausch von Wissen fördern.